### PR TITLE
Dev model

### PR DESCRIPTION
**Pull request by GRCaaaa**
## 新增内容
   锂离子固态电解质材料相关项目
   - 添加了完整的 XRDMatch 示例，用于 XRD 数据的半监督学习
   - 实现了 FlexMatch 算法，支持材料分类
   - 添加了 VGG 网络架构支持
   - 包含完整的代码、配置文件和文档

### CODE REVIEW

**Comment by HydrogenSulfate** [comment]
@GRCaaaa 你好，感谢你提交PR，我这里发现这个PR好像文件的数量有点问题，你那里是正常的吗？
image

**Review by HydrogenSulfate — COMMENTED** [review]
你好，请使用pre-commit格式化你的代码

Image

**Comment by leeleolay** [comment]
代码辛苦检查下，有太多额外的文件了

File: competition/IJCAI_2024_CAR
Comment (HydrogenSulfate) [inline competition/IJCAI_2024_CAR:1]: 无关的内容请还原

File: jointContribution/DU_CNN/requirements.txt
Comment (HydrogenSulfate) [inline jointContribution/DU_CNN/requirements.txt:1]: jointcontribution是无关内容，请还原

File: ppsci/externals/Open3D
Comment (HydrogenSulfate) [inline ppsci/externals/Open3D:1]: 无关内容请还原

File: examples/ai4materials/xrdmatch/loss/ce_loss.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@register
+class CrossEntropyLoss(nn.Layer):
+    def __init__(self, weight=None, ignore_index=-100, reduction="mean"):
+        super().__init__()
+        self.loss_fn = nn.CrossEntropyLoss(
+            weight=weight, ignore_index=ignore_index, reduction=reduction
+        )
+
+    def forward(self, logits, targets):
+        return self.loss_fn(logits, targets)
```
Comment (HydrogenSulfate) [inline examples/ai4materials/xrdmatch/loss/ce_loss.py]: 这看起来并不是ppsci的代码

**Review by HydrogenSulfate — COMMENTED** [review]
无关的内容请还原

**Comment by leeleolay** [comment]
@GRCaaaa  辛苦再检查下代码

File: competition/IJCAI_2024_CAR
Comment (leeleolay) [inline competition/IJCAI_2024_CAR:1]: 请还原

File: examples/xrdmatch/configs/xrdmatch.yaml
Comment (leeleolay) [inline examples/xrdmatch/configs/xrdmatch.yaml:1]: exampales下直接使用案例名，去掉子目录【ai4materials】

File: jointContribution/DU_CNN/requirements.txt
Comment (leeleolay) [inline jointContribution/DU_CNN/requirements.txt:1]: 无关文档请还原

File: ppsci/externals/Open3D
Comment (leeleolay) [inline ppsci/externals/Open3D:1]: 无关请还原

**Review by leeleolay — CHANGES_REQUESTED** [review]
需要改动的地方较多，辛苦仔细检查下，包括数据集，预训练模型权重文件是否需要上传到网上，辛苦提供原始文件的下载链接，我上传到云盘上提供链接可粘贴到文档中

**Comment by leeleolay** [comment]
是否已经修改完成？我看提交的pr里面文件还是有多余的
